installer/datafiles/base_omsagent.data

```diff
@@ -54,6 +54,7 @@ MAINTAINER:              'Microsoft Corporation'
 /opt/microsoft/omsagent/plugin/zabbixapi.rb;                            source/code/plugins/zabbixapi.rb;                      744; root; root
 /opt/microsoft/omsagent/plugin/oms_configuration.rb;                    source/code/plugins/oms_configuration.rb;              744; root; root
 /opt/microsoft/omsagent/plugin/out_oms_blob.rb;                         source/code/plugins/out_oms_blob.rb;                   744; root; root
+/opt/microsoft/omsagent/plugin/out_oms_api.rb;                          source/code/plugins/out_oms_api.rb;                    744; root; root
```

Comment:
It's a common output plugin. There is no default configuration yet. 
Customer can chain any input plugin to this output plugin to upload dynamic object to OMS as long as the output of the input plugin is a simple hash object.